scripts/e2e-custom.sh

```diff
@@ -257,6 +288,31 @@ test_taxonomy() {
     ilab taxonomy diff "${DIFF_ARGS[@]}"
 }
 
+test_convert_taxonomy_for_rag() {
+    step Convert user taxonomy for ingestion into vector store
+    ilab rag convert
+    task Taxonomy document conversion Complete
+}
+
+test_convert_user_document_for_rag() {
+    step Convert documents for ingestion into vector store
+    mkdir -p "${DATA_HOME}/instructlab/converted_documents"
+    ilab rag convert --input-dir "${DATA_HOME}/instructlab/raw_documents" --output-dir="${DATA_HOME}/instructlab/converted_documents"
```

Comment:
I can't find the `raw_documents` folder in the temp dir, maybe we missed to copy it from the `test-data` folder?